CHANGES.rst

```diff
@@ -8,6 +8,22 @@ photom
   flux conversion factors get computed correctly for both point-source and
   extended-source data. [#7019]
 
+tweakreg
+--------
+
+- Renamed ``gaia`` suffix to ``abs`` for the absolute astrometry
+  parameters. Specifically, ``gaia_catalog``->``abs_refcat``,
+  ``save_gaia_catalog``->``save_abs_catalog``. [#7023]
+
+- Removed ``align_to_gaia`` boolean step parameter since it now can be
+  indicated via ``abs_refcat`` parameter. When ``abs_refcat`` is None or an
+  empty string, alignment to an absolute astrometric catalog will be turned
+  OFF. When ``abs_refcat`` is a non-empty string, alignment to an absolute
+  astrometric reference catalog will be turned ON.
+
+- Bug fix: removed ``min_gaia`` which should have been removed in the
+  PR 6987. [#2023]
```

Comment:
typo here in the PR number: 2023 should be 7023

Reply:
either a typo or astigmatism